pom.xml

```diff
@@ -59,6 +59,7 @@
     <version.slf4j>1.7.30</version.slf4j>
     <version.solr>8.8.2</version.solr>
 
+    <version.fmt-maven-plugin>2.9.1</version.fmt-maven-plugin>
```

Comment:
Hier gäbe es schon Version `2.10`.

Reply:
Das braucht aber Java 11 und das Plugin supported explizit noch Java 8 bis auf weiteres.